service/upload/upload.go

```diff
@@ -1,12 +1,15 @@
 package uploadService
 
 import (
+	"regexp"
 	"strings"
 
 	"github.com/NyaaPantsu/nyaa/config"
 	"github.com/NyaaPantsu/nyaa/model"
 )
 
+const trackerRegex = `^(http[s]*|udp)://[a-z0-9\.:\-/?]+$`
```

Comment:
How are trackers stored in the db? If done safely then this regex should not be needed

Here's an example: `http://[2001:db8::1]:6969/announce?a=1&b=1?`